src/Components/test/testassets/BasicTestApp/Index.razor

```diff
@@ -1,8 +1,10 @@
 @using Microsoft.AspNetCore.Components.RenderTree
 <div id="test-selector">
   Select test:
-    <select @bind=SelectedComponentTypeName>
+    @* The id below is needed for some Ignitor tests *@
+    <select id="test-selector-select" @bind=SelectedComponentTypeName>
         <option value="none">Choose...</option>
+        <option value="BasicTestApp.ReliabilityComponent">Server reliability component</option>
```

Comment:
At some point we should reorder all these into alphabetical order. There are so many of them now that it bugs me each time I'm trying to find a specific one in the UI manually and I have to read the entire list from top to bottom!

Not saying you should do it now. But you *could*...

Reply:
I did this for the goodness of my heart